intel_realsense_payload_for_spotsim2real/IntelRealSenseCameraInterface.py

```diff
@@ -211,17 +213,24 @@ def blocking_capture(self):
                         )
                         pass
             else:
-                return (
-                    IntelRealSenseCameraInterface.DEPTH_FRAME,
-                    IntelRealSenseCameraInterface.CAPTURE_TIME,
-                )
+                try:
+                    return (
+                        IntelRealSenseCameraInterface.DEPTH_FRAME,
+                        IntelRealSenseCameraInterface.CAPTURE_TIME,
+                    )
+                except Exception as e:
+                    _LOGGER.warning(
+                        f"Unable to display the IntelRealSense images captured.{str(e)}"
+                    )
+                    os.kill(os.getpid(), signal.SIGINT)  # Kill the entire code
+                    # pass
```

Comment:
@Achuthankrishna can you actually see this string message displayed on terminal when intelRS gets disconnected?

Also, sig killing this code will mainly stop the intelRS service running on NUC / SpotCore right?

Reply:
Yes this just kills the payload script and the docker that runs the payload. So user needs to restart the payload, connect realsense and get back the feed.